src/main/resources/google_checks.xml

```diff
@@ -124,7 +124,8 @@
                     NOT_EQUAL, PLUS, PLUS_ASSIGN, QUESTION, RCURLY, SL, SLIST, SL_ASSIGN, SR,
                     SR_ASSIGN, STAR, STAR_ASSIGN, LITERAL_ASSERT, TYPE_EXTENSION_AND"/>
       <message key="ws.notFollowed"
-              value="WhitespaceAround: ''{0}'' is not followed by whitespace. Empty blocks may only be represented as '{}' when not part of a multi-block statement (4.1.3)"/>
+              value="WhitespaceAround: ''{0}'' is not followed by whitespace. Empty blocks may only
+               be represented as '{}' when not part of a multi-block statement (4.1.3)"/>
```

Comment:
how this is related to issue?

Reply:
@strkkk @romani It is not related to this issue but this is giving me an error, So I created a new commit for this formatting.